examples/player/simid_player.js

```diff
@@ -440,11 +467,14 @@ class SimidPlayer {
   
   /** The creative wants to play video. */
   onRequestPlay(incomingMessage) {
-    this.adVideoElement_.play().then(
-      // The play function returns a promise.
-      this.simidProtocol.resolve(incomingMessage),
-      this.simidProtocol.reject(incomingMessage)
-    );
+    
+    if (this.isLinearAd_) {
+      this.adVideoElement_.play()
+      .then(() => this.simidProtocol.resolve(incomingMessage))
+      .catch(() => this.simidProtocol.reject(incomingMessage));
+    } else {
+      this.simidProtocol.reject(incomingMessage);
```

Comment:
You need to reject non linear requests.

if (!this.isLinearAd_) {
errorMessage = {
errorCode : 1102,
message: 'non linear ads do not play video'
}
this.simidProtocol.reject(incomingMessage, errorMessage);